charmhelpers/contrib/ansible/__init__.py

```diff
@@ -139,6 +139,42 @@ def install_ansible_support(from_ppa=True, ppa_location='ppa:rquillo/ansible'):
 
 
 def apply_playbook(playbook, tags=None, extra_vars=None):
+    """Run a playbook.
+
+    This helper runs a playbook with juju state variables as context,
+    therefore variables set in application config can be used directly.
+    List of tags (--tags) and dictionary with extra_vars (--extra-vars)
+    can be passed as additional parameters.
+
+    Read more about playbook `_variables`_ online.
+
+    .. _variables: https://docs.ansible.com/ansible/2.5/user_guide/playbooks_variables.html
```

Comment:
Should be latest upstream docs: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html